module_utils/kafka_lib_acl.py

```diff
@@ -62,40 +62,16 @@ def process_module_acls(module, params=None):
                     'instead'
                 )
 
-        if len(acls) > 1:
-            acl_resource = ACLResource(
+        acl_resources_found = manager.describe_acls(
```

Comment:
I'm not sure we need to get all ACLs when kafka_acl module is called. When you have many ACLs it might be very slow to retrieve them.

Maybe a more elegant solution might be to extract the `if` directly inside callers to pass it to `process_module_acls`? What do you think of it? @steppi91 @StephenSorriaux ?